source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/files/product-artifact.tsx

```diff
@@ -47,7 +47,7 @@ class ProductArtifact extends React.Component<IProps> {
 
     return (
       <div className='product-artifact w-100 m-b-lg'>
-        <div className='header flex align-items-center'>
+        <div className='header flex align-items-center p-t-sm p-b-sm p-l-md p-r-md'>
```

Comment:
the utility classes enforce consistency much better than manually typing out px values :D